source/_drafts/The-Importance-of-Metadata.md

```diff
@@ -0,0 +1,18 @@
+---
+title: The Importance of Metadata
```

Comment:
title: 'Don't forget the importance of metadata'

Reply:
Oh yes, That is one nice title